cg/scripts/deploy-blockchain-contract.js

```diff
@@ -1,14 +1,12 @@
+const contractABI = require('../src/utils/blockchain/contract-abi.json');
```

Comment:
I prefer to use camel case even for abbreviations, so like:
`const contractABI` => `const contractAbi`

any preference @Jakedavies?

Reply:
oks, changing it